setup.py

```diff
@@ -62,7 +62,7 @@
         ],
         "console_scripts": ["pl-device-test=pennylane.devices.tests:cli"],
     },
-    "description": "PennyLane is a Python quantum machine learning library by Xanadu Inc.",
+    "description": "PennyLane is a cross-platform Python library for differentiable programming of quantum computers. Train a quantum computer the same way as a neural network.",
```

Comment:
```suggestion
    "description": "PennyLane is a cross-platform Python library for quantum computing, quantum machine learning, and quantum chemistry. Train a quantum computer the same way as a neural network.",
```

Reply:
even that positioning is out of date

Reply:
Just a tweak on the last suggestion:
```suggestion
    "description": "PennyLane is a quantum differentiable programming library for quantum computing, quantum machine learning, and quantum chemistry.",
```
Note that "cross platform" is missing. I personally have never found "cross platform" that exciting when I hear it, but others might think differently. For me, being cross platform is a necessity for me to use it, rather than a nice feature.

Reply:
> How about?
> 
> Just thinking if we want to keep the 'differentiable programming' keyword 🤔

this just doesn't read very well :thinking: